activesupport/test/cache/behaviors/cache_store_behavior.rb

```diff
@@ -131,9 +131,9 @@ def test_read_multi_with_expires
   end
 
   def test_read_multi_with_empty_keys_and_a_logger_and_no_namespace
-    @cache.options[:namespace] = nil
```

Comment:
This was the main offending line - we reset the namespace, and `@cache.clear` in `teardown` just flushed the db and we get nils in our tests.